### PR TITLE
Rename BankAccount address to billingAddress

### PR DESCRIPTION
**Pull request by ashkarpetin**
Rename request body schema which was missed in https://github.com/Rebilly/api-definitions/pull/91

### CODE REVIEW

**Review by subenson — DISMISSED** [review]

**Review by iamdevice — DISMISSED** [review]

**Comment by ashkarpetin** [comment]
php please re-review

File: openapi/components/schemas/BankAccounts/AccountNumberTypes/BBANType.yaml [hunk 1]
```diff
@@ -10,32 +10,32 @@ allOf:
...
           - checking
           - savings
           - other
       bankName:
-        description: Bank's name.
+        description: Bank's name
         type: string
       bic:
-        description: Bank Identifier Code.
+        description: Bank Identifier Code
```
Comment (vykhrystiuk) [inline openapi/components/schemas/BankAccounts/AccountNumberTypes/BBANType.yaml:35]:
```suggestion
        description: Bank identifier code
```
?

Comment (ashkarpetin) [reply openapi/components/schemas/BankAccounts/AccountNumberTypes/BBANType.yaml:35]: ?
https://www.theswiftcodes.com/

File: openapi/components/schemas/BankAccounts/AccountNumberTypes/IBANType.yaml [hunk 2]
```diff
@@ -8,25 +8,25 @@ allOf:
...
           in [SWIFT IBAN
           Registry](https://www.swift.com/standards/data-standards/iban).
         type: string
       bankName:
-        description: Bank's name.
+        description: Bank's name
         type: string
       bic:
-        description: Bank Identifier Code.
+        description: Bank Identifier Code
```
Comment (vykhrystiuk) [inline openapi/components/schemas/BankAccounts/AccountNumberTypes/IBANType.yaml:26]: https://github.com/Rebilly/api-definitions/pull/92/files#r433135258

Comment (ashkarpetin) [reply openapi/components/schemas/BankAccounts/AccountNumberTypes/IBANType.yaml:26]: ?
https://en.wikipedia.org/wiki/ISO_9362

**Review by vykhrystiuk — APPROVED** [review]

**Review by slavcodev — APPROVED** [review]

**Pull request merged**